forwarding/test_forwarded_hdr.py

```diff
@@ -0,0 +1,187 @@
+"""
+Tests for validate Forwarded header.
+"""
+from helpers import chains
+from framework import tester
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2022 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class TestForwaded(tester.TempestaTest):
+
+    backends = [
+        {
+            'id' : 'backend1',
+            'type' : 'deproxy',
+            'port' : '8000',
+            'response' : 'static',
+            'response_content' :
+            'HTTP/1.1 200 OK\r\n'
+            'Content-Length: 0\r\n\r\n'
+        }
+    ]
+
+    tempesta = {
+        'config' :
+        """
+        srv_group grp1 {
+        server ${server_ip}:8000;
+        }
+        vhost app {
+        proxy_pass grp1;
+        }
+        http_chain {
+        -> app;
+        }
+        """
+    }
+
+    clients = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'addr' : "${tempesta_ip}",
+            'port' : '80'
+        }
+    ]
+
+    req_params = []
+
+    response_status = '200'
+
+    def setUp(self):
+        tester.TempestaTest.setUp(self)
+
+    def start_all(self):
+        self.start_all_servers()
+        self.start_tempesta()
+        self.start_all_clients()
+        self.deproxy_manager.start()
+        self.assertTrue(self.wait_all_connections())
+        
+    def prepare_request(self, req_param):
+        req = ('GET / HTTP/1.1\r\n'
+               'Host: localhost\r\n'
+               'Forwarded: %s\r\n\r\n' % req_param)
+
+        return req
+
+    def client_send_req(self, client, req):
+        curr_responses = len(client.responses)
+        client.make_requests(req)
+        client.wait_for_response(timeout=1)
+        self.assertEqual(curr_responses + 1, len(client.responses))
+        
+        return client.last_response
+
+    def _test(self):
+        self.start_all()
+        client = self.get_client('deproxy') 
+        for param in self.req_params:
+            req = self.prepare_request(param)
+            resp = self.client_send_req(client, req)
+            self.assertEqual(resp.status, self.response_status)
+            client.restart()
+
+class SingleTest(object):
+    def test(self):
+        self._test()
```

Comment:
Please consider this comment https://github.com/tempesta-tech/tempesta-test/pull/221#discussion_r883928828 - we'll need a unified helper for this.

Reply:
> Please consider this comment [#221 (comment)](https://github.com/tempesta-tech/tempesta-test/pull/221#discussion_r883928828) - we'll need a unified helper for this.

ok, I am waiting for helper and will update my tests.